### PR TITLE
Find Similar Incidents - support new line

### PR DESCRIPTION
**Pull request by erezh31**
fixes https://github.com/demisto/etc/issues/18059

### CODE REVIEW

**Comment by erezh31** [comment]
@Itay4 @anara123 plz review

**Comment by Itay4** [comment]
@kirbles19 please review the md files

File: Releases/LatestRelease/FindSimilarIncidents.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Fix bugs:
```
Comment (Itay4) [inline Releases/LatestRelease/FindSimilarIncidents.md]: please change to be as documented here https://github.com/demisto/internal-content/tree/master/documentation/change-log

File: Releases/LatestRelease/FindSimilarIncidentsV2.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Fix bugs:
```
Comment (Itay4) [inline Releases/LatestRelease/FindSimilarIncidentsV2.md]: please change to be as documented here https://github.com/demisto/internal-content/tree/master/documentation/change-log

**Comment by kirbles19** [comment]
@erezh31 Done.

**Comment by erezh31** [comment]
@Itay4 please approve and merge

**Pull request merged**